speculation-rules.bs

```diff
@@ -517,9 +547,10 @@ A <dfn>prerender candidate</dfn> is a [=struct=] with the following [=struct/ite
   1. [=list/For each=] |prefetchCandidate| of |prefetchCandidates|:
     1. The user agent may run the following steps:
       1. Let |prefetchRecord| be a new [=prefetch record=] whose [=prefetch record/URL=] is |prefetchCandidate|'s [=prefetch candidate/URL=], [=prefetch record/anonymization policy=] is |prefetchCandidate|'s [=prefetch candidate/anonymization policy=], and [=prefetch record/label=] is "`speculation-rules`".
+        <p class="issue">Pass along |prefetchCandidate|'s [=prefetch candidate/referrer policy=].<p>
```

Comment:
I guess we're waiting on #212 before we tackle this?

Reply:
Yeah, to sketch the remaining plumbing:
- Add a field to prefetch records
- Set the prefetch algorithm's new document state's request referrer policy

The change to the existing prefetch spec would be different, so we may as well wait for that.

Reply:
PR #234 handles this.